test/routes/actions.test.js

```diff
@@ -13,14 +12,14 @@ function initContext(schemas, smartActionHookGetResponse) {
     .addInstance('stringUtils', {
       parameterize: jest.fn((name) => name),
     })
-    .addInstance('schemasGenerator', { schemas })
-    .addInstance('smartActionHookService', { getResponse: smartActionHookGetResponse })
-    .addInstance('smartActionHookDeserializer', new SmartActionHookDeserializer())
-    .addClass(ActionsRoutes)
+    .addInstance('schemasGenerator', () => ({ schemas }))
+    .addInstance('smartActionHookService', () => ({ getResponse: smartActionHookGetResponse }))
+    .addInstance('smartActionHookDeserializer', () => new SmartActionHookDeserializer())
+    .addUsingClass('actions', () => ActionsRoutes)
```

Comment:
Feels weird to me that we inject routes in the context here in test, bu not in the actual production code in `src/index.js`

Reply:
I agree, I just switched the previous context handling by the new one without too much thinking.
It does look weird indeed, but was already here for code refactoring reason (As it was injected then recovered by this `initContext` function.

I would be more confortable leaving this as is, then eventually refactor this specific test to remove it to prevent mixing the context library upgrade with some refactor

Reply:
👍 👌